### DISCUSSION TITLE
Restore Open File Explorer Windows After Update

### DISCUSSION THREAD

**Discussion by fanningmf** [valinet/ExplorerPatcher#3581] [Ideas]
I tend to have a lot of file explorer windows open throughout my work day.  Would it be possible for ExplorerPatcher to re-open these same windows after it has finished updating, because right now it essentially kills explorer.exe and those windows are gone.

Thank you.

**Comment by pyrates999** [comment]
Open up windows explorer.  Go to tools -> Folder Options.  Click on view tab and tick this:

![image](https://github.com/user-attachments/assets/626d54d2-0fe4-4d75-b423-20fafed307a5)

Then click the OK button.

**Reply by fanningmf** [reply]
That's Microsoft Windows folder options (at "log-on").  I already have that checked and that has nothing to do with ExplorerPatcher.  When ExplorerPatcher finishes an update, it kills the explorer.exe task (same as if you did so in Task Manager) which closes all the currently open File Explorer windows.  I'm asking for an enhancement to ExplorerPatcher to get a list of currently open windows and re-open them after the update completes.

**Reply by pyrates999** [reply]
Please wait for a developer to respond.
